site/docs/v1/tech/apis/identity-providers/twitch.adoc

```diff
@@ -0,0 +1,34 @@
+---
+layout: doc
+title: Twitch Identity Provider APIs
+description: APIs for creating, retrieving, updating and disabling the Twitch identity provider
+---
+
+== Overview
+
+[NOTE.since]
+====
+This API has been available since 1.28.0
+====
+
+The Google identity provider type will use the Google OAuth v2.0 login API. It will also provide a `Login with Google` button on FusionAuth’s login page that will direct a user to the Google login page.
```

Comment:
`Google` -> `Twitch`

Reply:
Oops.  Copy-paste error, but it's fixed now.